diff --git a/src/projects/compiler-rt/make/platform/clang_darwin.mk b/src/projects/compiler-rt/make/platform/clang_darwin.mk
index a297939..f76efbc 100644
--- a/src/projects/compiler-rt/make/platform/clang_darwin.mk
+++ b/src/projects/compiler-rt/make/platform/clang_darwin.mk
@@ -21,14 +21,9 @@ UniversalArchs.eprintf := i386
 Configs += 10.4
 UniversalArchs.10.4 := i386 x86_64
 
-# Configuration for targetting armv6. We need a few additional functions which
-# must be in the same linkage unit.
-Configs += armv6
-UniversalArchs.armv6 := armv6
-
 # Configuration for use with kernel/kexts.
 Configs += cc_kext
-UniversalArchs.cc_kext := armv6 armv7 i386 x86_64
+UniversalArchs.cc_kext := i386 x86_64
 
 ###
 
diff --git a/src/tools/clang/runtime/Makefile b/src/tools/clang/runtime/Makefile
index 918a3de..289d603 100644
--- a/src/tools/clang/runtime/Makefile
+++ b/src/tools/clang/runtime/Makefile
@@ -42,7 +42,7 @@ ifeq ($(shell test -d $(COMPILERRT_SRC_ROOT) && echo OK),OK)
 RuntimeDirs :=
 ifeq ($(OS),Darwin)
 RuntimeDirs += darwin
-RuntimeLibrary.darwin.Configs = eprintf 10.4 armv6 cc_kext
+RuntimeLibrary.darwin.Configs = eprintf 10.4 cC_kext
 
 # On Darwin, fake Clang into using the iOS assembler (since compiler-rt wants to
 # build ARM bits).
