diff --git a/src/projects/compiler-rt/lib/Makefile.mk b/src/projects/compiler-rt/lib/Makefile.mk
index d5545f1..111d63a 100644
--- a/src/projects/compiler-rt/lib/Makefile.mk
+++ b/src/projects/compiler-rt/lib/Makefile.mk
@@ -11,10 +11,7 @@ ModuleName := builtins
 SubDirs :=
 
 # Add arch specific optimized implementations.
-SubDirs += i386 ppc x86_64 arm
-
-# Add ARM64 dir.
-SubDirs += arm64
+SubDirs += i386 x86_64
 
 # Add other submodules.
 SubDirs += asan
diff --git a/src/projects/compiler-rt/make/platform/clang_darwin.mk b/src/projects/compiler-rt/make/platform/clang_darwin.mk
index c48d9ba..44895d6 100644
--- a/src/projects/compiler-rt/make/platform/clang_darwin.mk
+++ b/src/projects/compiler-rt/make/platform/clang_darwin.mk
@@ -55,11 +55,6 @@ UniversalArchs.eprintf := $(call CheckArches,i386,eprintf)
 Configs += 10.4
 UniversalArchs.10.4 := $(call CheckArches,i386 x86_64,10.4)
 
-# Configuration for targetting iOS for a couple of functions that didn't
-# make it into libSystem.
-Configs += ios
-UniversalArchs.ios := $(call CheckArches,i386 x86_64 armv7,ios)
-
 # Configuration for targetting OSX. These functions may not be in libSystem
 # so we should provide our own.
 Configs += osx
@@ -67,18 +62,11 @@ UniversalArchs.osx := $(call CheckArches,i386 x86_64,osx)
 
 # Configuration for use with kernel/kexts.
 Configs += cc_kext
-UniversalArchs.cc_kext := $(call CheckArches,armv7 i386 x86_64,cc_kext)
-
-# Configuration for use with kernel/kexts for iOS 5.0 and earlier (which used 
-# a different code generation strategy).
-Configs += cc_kext_ios5
-UniversalArchs.cc_kext_ios5 := $(call CheckArches,x86_64 armv7,cc_kext_ios5)
+UniversalArchs.cc_kext := $(call CheckArches,i386 x86_64,cc_kext)
 
 # Configurations which define the profiling support functions.
 Configs += profile_osx
 UniversalArchs.profile_osx := $(call CheckArches,i386 x86_64,profile_osx)
-Configs += profile_ios
-UniversalArchs.profile_ios := $(call CheckArches,i386 x86_64 armv7,profile_ios)
 
 # Configurations which define the ASAN support functions.
 #Configs += asan_osx_dynamic
diff --git a/src/tools/clang/runtime/compiler-rt/Makefile b/src/tools/clang/runtime/compiler-rt/Makefile
index 930a8c2..08bad7e 100644
--- a/src/tools/clang/runtime/compiler-rt/Makefile
+++ b/src/tools/clang/runtime/compiler-rt/Makefile
@@ -80,8 +80,8 @@ RuntimeDirs :=
 ifeq ($(OS),Darwin)
 RuntimeDirs += darwin
 RuntimeLibrary.darwin.Configs := \
-	eprintf.a 10.4.a osx.a ios.a cc_kext.a cc_kext_ios5.a \
-	profile_osx.a profile_ios.a
+	eprintf.a 10.4.a osx.a cc_kext.a \
+	profile_osx.a
 #	Apple Internal: Disable building asan libraries. <rdar://12423266>
 #	asan_osx_dynamic.dylib ubsan_osx.a
 
