Index: include/clang/Sema/Sema.h
===================================================================
--- include/clang/Sema/Sema.h	(revision 295545)
+++ include/clang/Sema/Sema.h	(working copy)
@@ -1054,7 +1054,7 @@
   Sema(Preprocessor &pp, ASTContext &ctxt, ASTConsumer &consumer,
        TranslationUnitKind TUKind = TU_Complete,
        CodeCompleteConsumer *CompletionConsumer = nullptr);
-  ~Sema();
+  virtual ~Sema();
 
   /// \brief Perform initialization that occurs after the parser has been
   /// initialized but before it parses anything.
@@ -1163,7 +1163,7 @@
 
   void emitAndClearUnusedLocalTypedefWarnings();
 
-  void ActOnEndOfTranslationUnit();
+  virtual void ActOnEndOfTranslationUnit();
 
   void CheckDelegatingCtorCycles();
 
@@ -3341,7 +3341,7 @@
     return FullExprArg(FE.get());
   }
 
-  StmtResult ActOnExprStmt(ExprResult Arg);
+  virtual StmtResult ActOnExprStmt(ExprResult Arg);
   StmtResult ActOnExprStmtError();
 
   StmtResult ActOnNullStmt(SourceLocation SemiLoc,
